docs/metrics/introduction/metric-formats.md

```diff
@@ -73,6 +73,10 @@ Where:
 * `value` is any numeric value.
 * `timestamp` is a UNIX timestamp.
 
+:::important
+Currently Sumo interprets meta tags as non-identifying dimensions. However, this is subject to a change and meta tags and intrinsic tags will both be treated as identifying dimensions. To conform to the target behavior from the beginning, place all your metric's metadata in `intrinsic_tags` section.
```

Comment:
I would just write what the current state is. Introducing the statement _meta tags and intrinsic tags will both be treated as identifying dimensions_ might be confusing since it's not currently the case.

Reply:
Applied @stacykor suggestion here.